File: src/main/java/no/nav/melosys/eessi/models/sed/medlemskap/impl/MedlemskapA002.java
package no.nav.melosys.eessi.models.sed.medlemskap.impl;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;
import no.nav.melosys.eessi.models.sed.medlemskap.Medlemskap;

@JsonIgnoreProperties(ignoreUnknown = true)
@Data
public class MedlemskapA002 extends Medlemskap {

    private UnntakA002 unntak;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/medlemskap/impl/MedlemskapA002.kt
package no.nav.melosys.eessi.models.sed.medlemskap.impl

import no.nav.melosys.eessi.models.sed.medlemskap.Medlemskap

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class MedlemskapA002(
    var unntak: UnntakA002? = null
) : Medlemskap
